### ISSUE TITLE
Apply sinc correction to fourier slice projections

### ISSUE THREAD

**Issue by michael-0brien** [mjo22/cryojax#396]
Need option to apply sinc correction to fourier projections computed with the `FourierSliceExtraction` method. This is the standard in other softwares; the reason is due to the details of linear interpolation in fourier space.

We have this implemented already in the `cryojax.image.operators.InverseSincFilter`; this should actually be a mask though, so need to fix that.
